src/main/java/dukepro/handlers/TasksManager.java

```diff
@@ -0,0 +1,152 @@
+package dukepro.handlers;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import dukepro.tasks.Task;
+
+/**
+ * Class for TasksManager.
+ */
+public class TasksManager {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private Storage storage;
+
+    /**
+     * Creates a TasksManager class.
+     *
+     * @return A TasksManager.
+     */
+    public TasksManager() {
+        this.storage = new Storage();
+        this.storage.readfile(this);
+    }
+
+    /**
+     * Adds a Task and update Storage.
+     *
+     * @param task A Task.
+     * @return A String.
+     */
+    public String addTask(Task task) {
+        //add to the tasks
+        tasks.add(task);
+        //write to file
+        storage.addTask(task);
```

Comment:
Also curious to why you would implement the storage of tasks like this and not save the entire tasklist after each user command in the main program?